packages/manager/modules/pci/src/components/chartjs/chartjs.directive.js

```diff
@@ -0,0 +1,56 @@
+import set from 'lodash/set';
+import uniqueId from 'lodash/uniqueId';
+import Chart from 'chart.js'; // eslint-disable-line
+
+import template from './chartjs.html';
+
+export default /* @ngInject */ () => ({
+  restrict: 'A',
```

Comment:
Should this be moved to common components folder so that it can be reused in other projects too? '/manager/packages/manager/modules/manager-components' seems to be right place.

Reply:
I think it's a good idea but wouldn't it be better to have this kind of components in the ui-kit instead of the manager ?

Reply:
All the components in UI-KIT are public, reused across Manager, web, order, and designed, approved by OVHcloud UX/design team. "manager-components" module is used to store all components reused in Manager. As Ganesh pointed, please prefer using AngularJs charts. This library is used in Manager and it would be ideal, simple and efficient to use it again.

Reply:
PU has moved some of these components to manager-components module through another PR